recipes-kernel/linux/5.15/patches/blktap2.patch

```diff
@@ -1057,26 +1054,20 @@ PATCHES
 +	del_gendisk(gd);
 +	gd->private_data = NULL;
 +
-+	cleanup_queue(gd->queue);
++	blk_cleanup_disk(gd);
++
++	blk_mq_free_tag_set(&tapdev->tag_set);
 +
-+	put_disk(gd);
 +	tapdev->gd = NULL;
 +
-+	mutex_lock(&tapdev->lock);
```

Comment:
Looks like here and elsewhere, the changes from https://github.com/OpenXT/xenclient-oe/pull/1438 were dropped.

(I have since repro-ed the crash https://github.com/OpenXT/xenclient-oe/pull/1438 tried to fix.  I think https://github.com/OpenXT/xenclient-oe/pull/1438 is still valid, but the trigger is different.)

Reply:
Yea that's a mistake on my part, I will amend the patch to keep #1438 included.

Reply:
The latest push should have your #1438 content included.